src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/SubmissionComponent.scala

```diff
@@ -107,6 +107,14 @@ trait SubmissionComponent {
       }))
     }
 
+    def countByStatus(workspaceContext: SlickWorkspaceContext): ReadAction[Map[String, Int]] = {
+      filter(_.workspaceId === workspaceContext.workspaceId).groupBy(s => s.status).map { case (status, submissions) =>
+        (status, submissions.length)
```

Comment:
Can you just do `(status -> submissions.length)` here and not have to subsequently turn it into a map?